tests/services/test_medication_reminder_scheduler.py

```
@@ -0,0 +1,257 @@
+
+    @pytest.mark.asyncio
+    async def test_end_date_today_still_fires(
+        self, db_session, patient_with_owner, mock_bus
+    ):
+        _make_med(
+            db_session,
+            patient_with_owner.id,
+            effective_period_end=date(2026, 6, 9),
+        )
+
+        _, due, _, published = await _run_tick(db_session, "08:00")
```

Ruff (0.15.15)

[warning] Unpacked variable `due` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)